api/guild_template.go

```diff
@@ -0,0 +1,56 @@
+package api
+
+import (
+	"time"
+
+	"github.com/DisgoOrg/restclient"
+)
+
+// GuildTemplate is a template used for copying guilds https://discord.com/developers/docs/resources/guild-template
+type GuildTemplate struct {
+	Disgo        Disgo
+	Code         string       `json:"code"`
+	Name         string       `json:"name"`
+	Description  *string      `json:"description,omitempty"`
+	UsageCount   int          `json:"usage_count"`
+	CreatorID    Snowflake    `json:"creator_id"`
+	Creator      *User        `json:"creator"`
+	CreatedAt    time.Time    `json:"created_at"`
+	UpdatedAt    time.Time    `json:"updated_at"`
+	GuildID      Snowflake    `json:"source_guild_id"`
+	PartialGuild PartialGuild `json:"serialized_source_guild"`
+	IsDirty      *bool        `json:"is_dirty,omitempty"`
```

Comment:
should this rly be a *bool instead of bool?
if its not present it should be false
```suggestion
	IsDirty       bool        `json:"is_dirty,omitempty"`
```

Reply:
I agree, the pointer should be removed